qiskit/ignis/logging/ignis_logging.py

```diff
@@ -0,0 +1,321 @@
+# -*- coding: utf-8 -*-
+
+# Copyright 2018, IBM.
+#
+# This source code is licensed under the Apache License, Version 2.0 found in
+# the LICENSE.txt file in the root directory of this source tree.
+
+"""
+Ignis Logging
+"""
+
+import logging
+import logging.handlers
+import logging.config
+from logging import Logger
+import os
+import glob
+from datetime import datetime
+
+
+class IgnisLogger(logging.getLoggerClass()):
+    """
+    A logger class for Ignis. IgnisLogger is a like any other logging.Logger
+    object except it has an additional method:    log_to_file, used to log data
+    in the form of key:value pairs to a log file. Logging configuration is
+    performed via a configuration file and is handled by IgnisLogging.
+
+    """
+    def __init__(self, name, level=logging.NOTSET):
```

Comment:
What are name and level here?

Reply:
These are standard Python logging params: name identifies the logger object within the logging package (by default logging messages are prefixed by the logger name), level is the verbosity level (e.g. debug/warning/error). 
I'll add a comment about it in the docstring.